pootle/static/js/shared/utils/i18n.js

```diff
@@ -44,9 +44,18 @@ function formatComponent(str, ctx) {
   return result;
 }
 
-export function t(string, ctx = null) {
+
+export function gettextComponentTemplate(string, ctx = null) {
   if (!ctx) {
     return gettext(string);
   }
   return formatComponent(gettext(string), ctx);
 }
+
+
+export function t(string, args) {
+  return interpolate(gettext(string), args, true);
+}
+
+
+export const tct = gettextComponentTemplate;
```

Comment:
FYI there's a [specific language syntax](http://www.2ality.com/2014/09/es6-modules-final.html) for this:`export { gettextComponentTemplate as tct};`. Although there's no reason not to do`export function tct(....)` directly and align with what's already done. I'll commit a change for this.